File: Guarana.java
import javax.swing.*;

public class Guarana extends Plant{

    Guarana(World world1, int x, int y) {
        SetStrength(0);
        SetInitiative(0);
        SetCords(x, y);
        SetMove(true);
        world = world1;
    }
    @Override
    public String getName() {
        return "GUARANA";
    }
    @Override
    public void Drawing() {
        ImageIcon icon = new ImageIcon("src/Icons/guarana.png");
        world.GetButton(GetCords()).setIcon(icon);
    }
    @Override
    public void Collision(Organism attacked) {
        attacked.SetStrength(attacked.GetStrength() + 3);
        world.GetOrganisms().remove(this,attacked);
        //!!!NOTIFICATIONS WHEN ANIMAL IS DELETED
    }
}

File: guarana.py
from plant import Plant

class Guarana(Plant):
    def __init__(self, world1, x, y):
        super().__init__()
        self.SetStrength(0)
        self.SetInitiative(0)
        self.SetCords(x, y)
        self.SetMove(True)
        self.world = world1

    def getName(self):
        return "GUARANA"
    
    def Drawing(self):
        self.world.buttons[self.GetCords().y -1][self.GetCords().x -1].configure(bg='#FFC0CB')
    
    def New(self, text, reference, x, y):
        self.world.GetOrganisms().add(Guarana(reference, x, y))    


    def Collision(self, attacked):
        attacked.SetStrength(attacked.GetStrength() + 3)
        self.world.GetOrganisms().remove(self, attacked)
        # !!!NOTIFICATIONS WHEN ANIMAL IS DELETED
